anti-analysis/obfuscation/obfuscated-with-babel-obfuscator.yml

```diff
@@ -4,7 +4,9 @@ rule:
     namespace: anti-analysis/obfuscation
     authors:
       - jakub.jozwiak@mandiant.com
-    scope: file
+    scopes:
+      static: file
+      dynamic: unspecified  # TODO upgrade manually, contains match
```

Comment:
usage of `match: compiled to the .NET platform` vs just `format: dotnet` appears inconsistent, TODO fix for all rules